test/viper/record.mo

```diff
@@ -1,30 +1,32 @@
 // @verify
 
 actor Record {
-  type R = { a: Int; b: Text; };
+  type R = { aa: Int; b: Text; };
 
-  let fld1: R = { a = 42; b = "abacaba" };
-  var fld2: R = { a = 42; b = "abacaba" };
+  let fld1: R = { aa = 42; b = "abacaba" };
+  var fld2: R = { aa = 42; b = "abacaba" };
 
-  let empty: R = { a = 0; b = "" };
+  let empty: R = { aa = 0; b = "" };
+
+  let aa: Int = 42; // A field to check that "moc" handles disambiguation in record fields
```

Comment:
Interesting :thinking:. If I name this field like `a` then it will conflict with `a` argument from `$concat` function. :wtf:

Reply:
Oh, yeah.
Viper kind have one name space for everything. So it's better to change argument names of `$concat` to something like `$concat$1` and `$concat$2`.

Reply:
Decided to change it later to not mess up commit history